hatch.toml

```diff
@@ -31,7 +31,7 @@ pre-install-commands = [
 ]
 
 [envs.integ.scripts]
-test = "pytest --xfail-tb --no-cov -vvv --numprocesses=1 {args:test/integ}"
+test = "pytest --xfail-tb --no-cov -vvv --numprocesses=0 {args:test/integ}"
```

Comment:
0 means run in the main process: https://pytest-xdist.readthedocs.io/en/stable/distribution.html

This enables live logging during test runs.